src/buffer/buffer.rs

```diff
@@ -893,25 +893,29 @@ mod tests {
     #[rstest]
     #[case(false, true, [false, false, true, true, true, true])]
     #[case(true, false, [true, true, false, false, false, false])]
-    fn merge_skip(#[case] one: bool, #[case] two: bool, #[case] expected: [bool; 6]) {
-        let mut one = Buffer::filled(
-            Rect {
+    fn merge_skip(#[case] skip_one: bool, #[case] skip_two: bool, #[case] expected: [bool; 6]) {
```

Comment:
Wow, this test is profoundly opaque.

Reply:
Looking at git blame… They were introduced with #215, so at least that's some context why that stuff exists.